src/Components/GrantsFinalizeShow.js

```diff
@@ -159,16 +229,15 @@ export default function GrantsFinalizeShow(props) {
           <dd>{grant.purpose}</dd>
           <dt>RFP URL</dt>
           <dd>{grant.rfp_url}</dd>
+          <p className="GrantsFinalizeShow__TotalWordCount">
+            Total word count: <span>{totalWordCount}</span>
+          </p>
         </dl>
       </section>
 
       <hr />
```

Comment:
actually wait the mockaframe has total word count in the hero section and outside
in hero: https://www.figma.com/file/kEoH9HCjEQHHtyo4xdyJqj/Boilerplate?node-id=504%3A2045
under: https://www.figma.com/file/kEoH9HCjEQHHtyo4xdyJqj/Boilerplate?node-id=504%3A628

Reply:
Eli and decided that it would be best over - in the hero section - so it stays associated with the block of grant info and not the set of sections, esp. since the sections also have individual word counts. I'll go through and fix it in the figma, but we'll stick with that.